docs/flow/modin/data_management/factories.rst

```diff
@@ -0,0 +1,19 @@
+Factories Module Description
+""""""""""""""""""""""""""""
+
+Brief description
+'''''''''''''''''
+Modin has many execution backends. Calling any DataFrame API function will end up in some backend-specific method. The responsibility of dispatching high-level API calls to backend-specific function lays on :doc:`QueryCompiler </flow/modin/backends/query_compiler.rst`. Building the `QueryCompiler` from external data is the role of the factories module. The mission of this module is to route IO function calls from the API level to its actual backend-specific implementations, which builds `QueryCompiler` of the corresponding backend.
```

Comment:
```suggestion
Modin has multiple execution backends. Calling any DataFrame API function will end up in some backend-specific method. The responsibility of dispatching high-level API calls to backend-specific function lays on `QueryCompiler` which definition is a role of the factories module. Another mission of this module is to route IO function calls from the API level to its actual backend-specific implementations, which build `Query Compiler` of the corresponding backend.
```

Reply:
Factories module role is to _build_ proper QueryCompiler, not to _define_ it. There is also no "another" mission of this module. Its only function is to dispatch IO calls to the backend-specific implementation, which results in a building of the QueryCompiler.

@vnlitvinov I'm calling you, to be a judge here)

Reply:
So, before a proper QueryCompiler is built/created a proper type of the QueryCompiler is defined in a proper IO class that was defined in a Factory. On "another", it is similar to "one more" mission of factories. So, I wouldn't put aside the Factories' role of defining proper type for QueryCompiler to be constructed.

cc @devin-petersohn

Reply:
@YarShev agree with you, changed this sentence